app-tracking-protection/vpn-impl/src/main/res/values/strings-vpn.xml

```diff
@@ -364,7 +364,7 @@
     <string name="atp_ManageRecentAppsProtectionEmpty">Apps with recently blocked tracking attempts will appear here</string>
 
     <!--  App Tracking Protection Remove Feature Dialog -->
-    <string name="atp_RemoveFeatureDialogTitle">Disable and Delete Data?</string>
+    <string name="atp_RemoveFeatureDialogTitle">Disable and delete data?</string>
```

Comment:
This change needs to be in a different PR to trigger translations for other languages